tools/debugging/stress_test_transfers.py

```diff
@@ -340,10 +360,15 @@ def wait_for_address_endpoint(base_url: str, retry_timeout: int) -> str:
     """Keeps retrying the `/address` endpoint."""
     while True:
         try:
-            return get_address(base_url)
-        except (requests.RequestException, requests.ConnectionError) as e:
-            print(f"Waiting for {base_url} error={e}")
-            gevent.sleep(retry_timeout)
+            address = get_address(base_url)
+            log.info(f"{address} finished restarting ready")
```

Comment:
```suggestion
            log.info(f"{address} finished starting and is ready")
```
It's not restarting in all cases, is it?

Reply:
it is restarting on all but the first case.